### DISCUSSION TITLE
"Free trial" on user account pages when you're on an active subscription

### DISCUSSION THREAD

**Discussion by sptv2reg** [plausible/analytics#2839] [Anything Else]
When our users enters their settings page, it will say that they are on a free trial (with upgrade link), even though we are on a paid subscription (50M). This causes some confusion:

<img width="662" alt="image" src="https://user-images.githubusercontent.com/103186488/231151503-239d0ae6-79b1-4cfd-b59b-6675d61572d4.png">

**Comment by metmarkosaric** [comment]
Thanks for the feedback @sptv2reg! We've made it like that so anyone can add their own sites and start their own subscription if they need. But yeah, for teams it might not be that necessary. We plan to improve that in the future.

Just to confirm that only the owner of a site needs to have an active subscription with Plausible. Viewers and admins don't need to have a paid subscription and can access the stats even without it so they can ignore any of those "trial" messages. The only time a trial would become active in those cases would be if they add a new site to Plausible that they themselves own.
